docs/_partials-common/ncs-install-golioth-firmware-sdk-for-unix.md

```diff
@@ -8,7 +8,7 @@ import FirmwareSdkVer from '!!raw-loader!/docs/_versions/golioth-firmware-sdk.md
     <CodeBlock language="console">
         { `cd ~` + "\n" }
         { `west init -m https://github.com/golioth/golioth-firmware-sdk.git --mr ${FirmwareSdkVer.replace(/(\n)/gm, "")} --mf west-ncs.yml ~/golioth-ncs-workspace` + "\n" }
-        { `cd golioth-ncs-workspace` + "\n" }
+        { `cd golioth-ncs-workspace/modules/lib/golioth-firmware-sdk` + "\n" }
         { `git submodule update --init --recursive` + "\n" }
         { `west update` }
```

Comment:
To double-check, it's correct to run `west update` from `~/golioth-ncs-workspace/modules/lib/golioth-firmware-sdk` or should the used execute `cd ~/golioth-ncs-workspace` and then `west update`?

Reply:
Either should work. West recursively searches parent folders for a `.west` folder, similar to git.

Reply:
The `west update` can be run from anywhere in the tree initialized by west so we should be okay.